apps/backoffice-v2/src/common/hooks/useHomeLogic/useHomeLogic.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentProps, useEffect } from 'react';
+import { DateRangePicker } from '@/common/components/molecules/DateRangePicker/DateRangePicker';
+import { useZodSearchParams } from '@/common/hooks/useZodSearchParams/useZodSearchParams';
+import { HomeSearchSchema } from '@/pages/Home/home-search-schema';
+import { t } from 'i18next';
+import { useAuthenticatedUserQuery } from '@/domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery';
+import { useLocale } from '@/common/hooks/useLocale/useLocale';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+export const useHomeLogic = () => {
+  const locale = useLocale();
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+  const [{ from, to }, setSearchParams] = useZodSearchParams(HomeSearchSchema);
+  const { data: session } = useAuthenticatedUserQuery();
+  const { firstName, fullName, avatarUrl } = session?.user || {};
+  const queryParams = from || to ? `?from=${from}&to=${to}` : '';
```

Comment:
This one is redundant. Overtime search query params will change, might as well pass the full `search` prop from `useLocation` like done in `Cases.Item.tsx`.